src/app/Header/components/MenuLeft/MenuLeft.jsx

```diff
@@ -1,41 +1,91 @@
 import React from 'react';
 import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars, faTimes } from '@fortawesome/free-solid-svg-icons';
 import Link from '../../../../components/Link';
+import useToggler from '../../../../hooks/useToggler';
 
 const Left = styled.ul`
   list-style: none;
   padding: 16px 0;
   margin: 0;
   display: flex;
   align-items: center;
+
+  @media (min-width: 320px) and (max-width: 1024px) {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    background: #DFCAA8;
+    padding: 0px;
+    width: 114px;
+    top: 45px;
+    left: ${({ showSlideMenu }) => (showSlideMenu ? 0 : '-100%')};
```

Comment:
```suggestion
    width: 0;
    top: ${({ showSlideMenu }) => (showSlideMenu ? 45px : '-100%')};
```
试试这样呢，或者明天早会问问龙哥有没有更好的方法